nixos/modules/services/networking/connman.nix

```diff
@@ -22,6 +27,23 @@ in {
         '';
       };
 
+      extraConfig = mkOption {
+        type = types.lines;
+        default = ''
+        '';
+        description = ''
+          Configuration lines appended to the generated connman configuration file.
+        '';
+      };
+
+      networkInterfaceBlacklist = mkOption {
+        type = types.str;
```

Comment:
Couldn't this easily be a list of strings on the Nix side? Might be a little easier to do unusual things with.